packages/protobuf-test/extra/example.proto

```diff
@@ -15,8 +15,11 @@
 syntax = "proto3";
 package docs;
 
+import "google/protobuf/timestamp.proto";
+
 message Example {
   string foo = 1;
   bool bar = 2;
   Example baz = 3;
+  google.protobuf.Timestamp created = 4;
```

Comment:
I think we may end up putting all possible field types in here over time. We might just as well use protobuf_test_messages.proto3.TestAllTypesProto3 right away, and keep this as a simple example.